### PR TITLE
@sym/prevprime: added prevprime (fixes #722)

### LINKED ISSUE #722: add `prevprime`
should be an easy fix, just copy nextprime.m and modify.

@nvs-abhilash interested?

### PR DESCRIPTION
**Pull request by nvs-abhilash**
Hope this is what you want, for #722

### CODE REVIEW

File: inst/@sym/prevprime.m [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+%%   @result{} (sym) [2  7  97  997  65521]  (1×5 matrix)
+%% @end group
+%% @end example
+%%
+%% @seealso{@@sym/isprime, @@sym/nextprime}
+%% @end defmethod
+
+
+function y = prevprime(x)
+
```
Comment (latot) [inline inst/@sym/prevprime.m]: Hi, Thx for this!, i think is missed something like this here:
```
if (nargin ~= 1)  print_usage () end
```

Comment (nvs-abhilash) [inline inst/@sym/prevprime.m]: I thought of doing error checks but I didn't see it in @sym/nextprime.m, So I assumed it is not required. Should I add it?

Comment (latot) [inline inst/@sym/prevprime.m]: mm, if we check this is all mixed, some files have it and others not, lets wait @cbm755

Comment (cbm755) [inline inst/@sym/prevprime.m]: Yes the print_usage should be added. Many functions are missing this.

File: inst/@sym/prevprime.m [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+%% @group
+%% prevprime(sym(3))
+%%   @result{} ans = (sym) 2
+%
+%% prevprime([sym(3) 10 100 1000 65530])
+%%   @result{} (sym) [2  7  97  997  65521]  (1×5 matrix)
+%% @end group
+%% @end example
+%%
+%% @seealso{@@sym/isprime, @@sym/nextprime}
```
Comment (cbm755) [inline inst/@sym/prevprime.m]: Nextprime should have its seealso changed. Maybe isprime too?

Comment (nvs-abhilash) [inline inst/@sym/prevprime.m]: Yes, will do that

**Comment by nvs-abhilash** [comment]
Yes it gives error, should I add a test for this error too?

**Comment by nvs-abhilash** [comment]
@cbm755 Please review the recent changes, and see if any more modifications are required or not.

File: inst/@sym/prevprime.m [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+end
+
+
+%!assert (isequal (prevprime(sym(3)), 2));
+%!assert (isequal (prevprime(sym(20)), 19));
+%!assert (isequal (prevprime(sym([3 5 10])), [2 3 7]));
+
+%!error(prevprime(sym(2)))
+%!error(prevprime(sym(-2)))
```
Comment (cbm755) [inline inst/@sym/prevprime.m]: should be:
````
%!error prevprime(sym(2))
````

File: inst/@sym/prevprime.m [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+%% @end group
+%% @end example
+%%
+%% @seealso{@@sym/isprime, @@sym/nextprime}
+%% @end defmethod
+
+
+function y = prevprime(x)
+
+  if (nargin ~= 1) print_usage () end
```
Comment (cbm755) [inline inst/@sym/prevprime.m]: please format this in the usual way (three lines)

**Comment by cbm755** [comment]
I notice these PRs come from your master branch.  You should use a new branch for each PR.

**Comment by nvs-abhilash** [comment]
Yah I recently felt the need of it. Will do the modifications.

**Comment by nvs-abhilash** [comment]
Should I open another PR, or should I update the commit in my master branch for the time being?

**Comment by nvs-abhilash** [comment]
I have made a new PR #729 which fixes the above issue, I had to open a new one because I shifted my work to a new branch.

**Pull request merged**